src/components/Board/Board.container.js

```diff
@@ -533,32 +533,12 @@ export class BoardContainer extends Component {
       speak,
       intl,
       boards,
-      showNotification
+      showNotification,
+      navigationSettings
     } = this.props;
     const hasAction = tile.action && tile.action.startsWith('+');
 
-    if (tile.loadBoard) {
-      try {
```

Comment:
@martinbedouret I saw that you added this `try` block in 92dfe6246235173900742e42b94b984e7fb0a3e1—presumably to handle an invalid board id. Is it still needed, since we now try defaulting to a board that matches the folder name, and show a notification if that doesn't work? if so, I'll put it back.

Reply:
Your implementation is right.
According to it, the find result will show the error in case the board is missed. Great code.